appcues/src/main/java/com/appcues/analytics/SdkMetrics.kt

```diff
@@ -0,0 +1,76 @@
+package com.appcues.analytics
+
+import java.util.Date
+import java.util.UUID
+
+internal data class SdkMetrics(
```

Comment:
I'm not sure if it would be cleaner if SdkMetric was an internal object, and we had a data called QualificationMetrics with these 4 properties.

Reply:
that way if in the future we decide to track other things we could create other data classes, and another set of functions to support that metrics

Reply:
ok, yeah I think that would be a pretty straightforward refactor like you said - if/when we have the need to expand things. I'm inclined to just keep the single use case for now since that's all I'm currently aware of needing.